public/modules/moduleMap.html

```diff
@@ -32,7 +31,7 @@
                 </button>
               </td>
             </tr>
-              <tr>
+<!--               <tr>
```

Comment:
Commented code isn't usually checked in.  Why has this been commented as opposed to deleted?